### PR TITLE
Expand setup docs.

### PR DESCRIPTION
**Pull request by mikalstill**
Just some quick tweaks to document what to install on Ubuntu 16.04 to get builds working.

### CODE REVIEW

**Comment by mikalstill** [comment]
This test failure looks unrelated, but I am not sure how to request a re-test...

File: README.md [hunk 1]
```diff
@@ -15,7 +17,15 @@ The followings are required for building Kilda controller:
  - Docker Compose 1.20.0+
  - GNU Make 4.1+
 
-You need to rise maven RAM limit at least up to 1G.
+On Ubuntu 16.04, you can install those dependancies like this:
```
Comment (rtretyak) [inline README.md]: typo `dependEncies`

File: README.md [hunk 2]
```diff
@@ -15,7 +17,15 @@ The followings are required for building Kilda controller:
...
+
+```
+apt-get install maven openjdk-8-jdk python python3 docker.io docker-compose virtualenv make
+```
```
Comment (PinGwynn) [inline README.md]: Or you can run:
```
base/hacks/ensure.buildtools.sh && base/hacks/load_kilda_dev_ubuntu.sh
```

File: README.md (hunk 1 above)
Comment (niksv) [reply README.md]: @mikalstill Please fix

**Pull request merged**